lib/hint.rb

```diff
@@ -67,8 +67,7 @@ def do_complete_inspection(options)
          "#{program_name} inspect #{options[:host]} --name #{options[:name]} --extract-files"
       elsif options[:docker_container]
         "To do a full inspection containing all scopes and to extract files run:\n" \
-         "#{program_name} inspect-container --docker #{options[:docker_container]} " \
-         "--name #{options[:name]} --extract-files"
+         "#{program_name} inspect-container --name #{options[:name]} --extract-files"
```

Comment:
You can't remove the `#{options[:docker_container]}` here, it's necessary for the command to work.